src/App.tsx

```diff
@@ -13,6 +13,8 @@ import { GlobalStyle } from './styles/GlobalStyle'
 import { Container, Content } from './styles/Main'
 import { defaultTheme } from './styles/theme'
 
+import '../i18n.js'
```

Comment:
Why are you using js instead of ts here?

Reply:
idk i will change

Reply:
I couldn't find the @types for i18next-fs-backend

Reply:
i18next-fs-backend not have a types